### ISSUE TITLE
FR: Add option to delete 'back of the note card' and the corresponding heading section in MD file

### ISSUE THREAD

**Issue by pikatwinky** [zsviczian/obsidian-excalidraw-plugin#2040]
### Is your feature request related to a problem? Please describe.

Most of the time i try to leverage this fantastic feature (back of the note card) inside the drawing view, cause i think its great to use the powerful obsidian editor features and at the same time being able to sketching without having to leave the scene, however when doing this and create some cards sometimes they're only just drafts that i have to delete at once, but with the current project status, i have to delete the card in Excalidraw view and manually have to delete once again the corresponding header section inside the MD view. 
When you released this amazing feature i claimed in the forum that [Excalidraw had been surpassed core Canvas plugin](https://forum.obsidian.md/t/canvas-drawing-and-pencil-or-pen-support/50245/92), but i still miss Canvas'es capability of creating and deleting cards without the user having to think about necessarily name them or manually deleting them (twice i mean)

### Describe the solution you'd like

Add a confirmation dialog when delete a card from the context menu asking if you want to delete the corresponding heading section as well. It could be great if this setting could be modifided in the plugin settings  

### Real-life usage scenarios

This feature could be make the experience of deleting cards smoother. 

### Describe alternatives you've considered

Deleting them manually, however as the number of cards increases this task turns to be a bit uncomfortable

**Comment by zsviczian** [comment]
yeah, this is a good point, I've been thinking about this myself as well.
